css/style.css

```diff
@@ -1 +1,1461 @@
+  fill: var(--secondary-color);
+}
+
+header .media-btn {
+  background-color: var(--secondary-color);
+  transition: color 0.3s ease-in-out;
+}
+
+header .media-btn:active {
+  background-color: rgb(35, 46, 57);
+  border: 2px solid var(--secondary-color);
+}
+
+.discord-btn {
+  width: 24px;
+  height: 24px;
+}
+
+.wallet-icon {
+  width: 24px;
+  height: 24px;
+}
+
+/**************** Hero section **************************/
+
+.hero-box {
+  padding-top: 141px;
+  background-image: url(../img/bg-s/star-big.svg),
+    linear-gradient(to top, rgba(4, 11, 17, 1), rgba(4, 11, 17, 0)),
+    radial-gradient(
+      ellipse at center,
+      rgba(48, 113, 101, 0.3) 10%,
+      rgba(20, 27, 34, 0.3) 70%
+    );
+  background-size: 496.05px auto, cover, 950px 278px;
+  background-position: bottom -50px right -170px, bottom, top -100px left -200px;
+  background-repeat: no-repeat;
+  background-color: var(--secondary-color);
+}
+
+.hero-content {
+  width: 750px;
+}
+
+.hero-title {
+  font-size: 72px;
+  line-height: 80px;
+}
+
+.hero-text {
+  font-family: var(--text-font);
+  font-size: 20px;
+  line-height: 30px;
+  margin: 20px 0 24px;
+}
+
+.hero-number {
+  font-size: 44px;
+  line-height: 56px;
+  text-shadow: 0px 4px 13px rgba(255, 255, 255, 0.4);
+}
+
+.hero-description {
+  font-family: var(--text-font);
+  font-size: 16px;
+  line-height: 26px;
+  color: var(--text-color);
+}
+
+.hero-btn {
+  margin-top: 48px;
+  gap: 24px;
+}
+
+/**********************  nft-cards section **************************/
+
+.nft-cards {
+  padding-top: 60px;
+  padding-bottom: 50px;
+  background-color: var(--main-background-color);
+  text-align: center;
+  display: flex;
+  gap: 30px;
+  overflow-x: auto;
+  white-space: nowrap;
+}
+
+.nft-card {
+  margin: 0 auto;
+  width: 330px;
+  height: 378px;
+  padding: 20px;
+  background-color: var(--article-bg-color);
+}
+
+.nft-card img {
+  width: 330px;
+  height: 330px;
+  margin: 0 auto;
+}
+
+.nft-card-title {
+  padding-top: 10px;
+  font-size: 20px;
+  line-height: 28px;
+  text-align: left;
+}
+
+/***************************** About section *******************************/
+
+.about {
+  padding-top: 50px;
+  padding-bottom: 50px;
+  text-align: center;
+  background-color: var(--main-background-color);
+}
+
+.about-description {
+  margin: 0 auto;
+  margin-top: 16px;
+  font-family: var(--text-font);
+  color: var(--text-color);
+  width: 930px;
+}
+
+.about-list {
+  margin-top: 60px;
+  list-style-position: inside;
+  text-align: left;
+}
+
+.about-list li {
+  margin: 0 auto;
+  padding: 30px;
+  background-color: var(--secondary-color);
+  margin-bottom: 30px;
+  width: 570px;
+  height: 224px;
+  box-sizing: border-box;
+  border-color: transparent;
+  transition: all 0.3s ease-in-out;
+}
+
+.about-list li::marker {
+  content: counter(list-item, decimal-leading-zero);
+  color: var(--secondary-color);
+  font-size: 80px;
+  line-height: 96px;
+  text-shadow: 0px 0px 1px var(--btn-primary-color);
```

Comment:
there is no point in using units if the value is 0:
`text-shadow: 0 0 1px var(--btn-primary-color);`

Reply:
#fixed